sfputil/main.py

```diff
@@ -688,6 +693,148 @@ def eeprom(port, dump_dom, namespace):
 
     click.echo(output)
 
+# 'eeprom-hexdump' subcommand
+@show.command()
+@click.option('-p', '--port', metavar='<port_name>', required=True, help="Display SFP EEPROM hexdump for port <port_name>")
+@click.option('-r', '--page', metavar='<page_number>', help="Display SFP EEEPROM hexdump for <page_number_in_hex>")
```

Comment:
Why '-r'? This doesn't seem intuitive. Since '-p' is already taken for the port I would prefer '-n' for page **n**umber.

Reply:
I have addressed this in the new commit now.